proto/prysm/v1alpha1/validator.proto

```diff
@@ -566,6 +567,9 @@ message BlockRequest {
 
     // Signal server to skip outsourcing block request from mev-boost/relayer so that returned block will always be a local block.
     bool skip_mev_boost = 4;
+
+    // Percentage multiplier to apply to the builder's payload value when choosing between a builder payload header and payload from the paired execution node
+    google.protobuf.UInt64Value builder_boost_factor = 5;
```

Comment:
I wanted to keep it nullable since 0 is a valid value and we don't have defaults in protobuf3. this way i can check if it was set or not and add a default